### ISSUE TITLE
Version flag

### ISSUE THREAD

**Issue by romainmenke** [google/guetzli#130]
Would it it be possible and useful for others than just me to add a version flag?
I wrote this little wrapper to manage compressions : [github.com/romainmenke/simple-guetzli](https://github.com/romainmenke/simple-guetzli)

It keeps a log of past compressions to make it work nicely with a watch folder. I thought it would be interesting to optionally recompress all images if a new version is released. To achieve this it would be awesome if there was a version flag.

**Comment by mgrhm** [comment]
I second this.  It would be useful for the above use-case, and for other utilities and wrappers for `guetzli`.

It would also make `guetzli` conform a little better to expected POSIX behaviour.  The `--version` and `--help` flags are nearly ubiquitous and including them is good practice.

**Comment by robryk** [comment]
When would you envision the version to increase? Only when a significant number of images will produce different output of on such releases as 1.0.1 too?

**Comment by romainmenke** [comment]
Current Releases already follow semantic versioning. I did not mean to question the logic behind the current versioning. I only suggested adding a flag that would return the currently installed version.

**Comment by mgrhm** [comment]
A flag like this would also be useful to determine which version of `guetzli` is installed if/when CUDA/GPU processing (#44) is introduced.

**Comment by gpant** [comment]
IMHO the actual version should not change unless the actual codebase changes significantly. In order to be able to identify the algorithm state and optimizations you can append the commit the build is based on.

ie guetzli 0.2.1 - c03e7261a5944b0b65043c0bb8b16c52b9329917

This way you can always build on a previous commit to get older algorithm versions and it can be tracked.

**Comment by romainmenke** [comment]
@gpant I agree with you on the logic behind the versioning but that is besides the point.
I opened this issue not to discus when to bump the version but to request adding a cli flag. 

You can open another issue to discus versioning itself.
